op-challenger/game/fault/preimages/large.go

```diff
@@ -38,3 +41,31 @@ func (p *LargePreimageUploader) UploadPreimage(ctx context.Context, parent uint6
 	// todo(proofs#467): once the challenge period is over, call `squeezeLPP` on the preimage oracle contract.
 	return errNotSupported
 }
+
+// initLPP initializes the large preimage proposal.
+// This method *must* be called before adding any leaves.
+func (p *LargePreimageUploader) initLPP(ctx context.Context, uuid *big.Int, partOffset uint32, claimedSize uint32) error {
```

Comment:
We probably should actually call this from `UploadPreimage`.  It's the first step in the process so we should be able to start testing and actually see this call succeed - obviously can't upload the whole preimage yet, but good to verify the pieces we're building actually work as we go instead of waiting until the end before testing it in practice.